src/bin/pg_autoctl/pgctl.c

```diff
@@ -52,7 +52,6 @@ static bool escape_recovery_conf_string(char *destination,
 static bool prepare_primary_conninfo(
 	char *primaryConnInfo,
 	int primaryConnInfoSize,
-	int primaryNodeId,
```

Comment:
This was added in #154. looks like a wrong removal

Reply:
The primaryNodeId was added as a parameter but unused in the body of the function, and I cleaned it up in the release branch, so now we are porting that fix back to master...